src/xenia/gpu/gl4/command_processor.cc

```diff
@@ -335,7 +335,7 @@ bool CommandProcessor::SetupGL() {
       "    }\n"
       "    EmitVertex();\n"
       "    EndPrimitive();\n"
-      "  } else {\n"
+      "  } else if (gl_in[0].gl_Position.y == gl_in[1].gl_Position.y) {\n"
```

Comment:
(this may be ok, as the hardware may cull rects not matching these formats - just curious if you'd seen any missing rects in your experimenting)